kolibri/plugins/learn/assets/src/views/LearnExamReportViewer.vue

```diff
@@ -96,8 +98,12 @@
         };
       },
       reportVisible() {
+        const quiz = this.activeClassesQuizzes.find(q => q.id === this.exam.id) || this.exam;
```

Comment:
This feels a little hacky - because of the semi-dependency on the maybe loaded, maybe not activeClassesQuizzes (which is only initialized if you have been on the home page).

But it also works in spite of my best attempt to break it - so it will do for now, until we can better align the loading of `this.exam` to be better aligned with what's in `this.activeClassesQuizzes`.

Reply:
I agree this is a bit hacky. Since the exam is cached and loaded from the state, initially I tried to remove this caching, but the exam's archive field still would not update to true when it should have. I could open an issue for the loading to be refactored so that it hopefully doesn't get lost!

Reply:
Please do! Can add it as a sub-issue of https://github.com/learningequality/kolibri/issues/11722 as the refactor should probably involve removing the use of vuex and shifting to using a composable!